tools/bookrunner/src/litani.rs

```diff
@@ -16,6 +16,7 @@ use std::process::{Child, Command};
 /// attributes in such files, but it may require you to
 /// extend it if advanced features are used (e.g., pools)
 #[derive(Debug, Deserialize)]
+#[allow(dead_code)]
```

Comment:
Following the previous Zyad's advice, why do we need to keep this code around with `dead_code` attribute? Maybe @adpaco-aws @jaisnan could tell us whether or not we should just delete it.

Reply:
I will create a separate PR to completely delete bookrunner.

Reply:
Agreed, the `#[allow(dead_code)]` annotations are OK here for now.